app/views/sponsors/sponsors_footer.html.haml

```diff
@@ -0,0 +1,5 @@
+%h3.section-title
+  Sponsors
+.sponsors-footer-wrapper
+  - @sponsors_in_footer.each do |sponsor|
+    = render "sponsor_footer", sponsor: sponsor
```

Comment:
Maybe a bit silly but good to know that you can do this here: 

```
= render "sponsor_footer", collection: @sponsors_in_footer, as: :sponsor
```
see https://guides.rubyonrails.org/layouts_and_rendering.html#local-variables

Reply:
I had to include 'partial' to get this to work. 
```
= render partial: "sponsor_footer", collection: @sponsors_in_footer, as: :sponsor
```
But I updated it.